app/models/issue.rb

```diff
@@ -1,8 +1,10 @@
 class Issue < ApplicationRecord
+  belongs_to :user
+  attr_accessor :attachment
```

Comment:
@igorkasyanchuk Removed attachment field from DB, we need attr_accessor here to upload our images to public/upload/issues folder